docs/source/developers/release_verification.rst

```diff
@@ -44,20 +44,36 @@ Linux and macOS
 In order to run the verification script either for the source release or the
 binary artifacts see the following guidelines:
 
+Required source verification
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Individuals are REQUIRED to download all signed source code packages onto their
+own hardware, validate all cryptographic signatures, compile as provided,
+and test the result on their own platform in order to cast a +1 vote.
+
 .. code-block::
 
    # this will create and automatically clean up a temporary directory for the verification environment and will run the source verification
    TEST_DEFAULT=0 TEST_SOURCE=1 verify-release-candidate.sh $VERSION $RC_NUM
```

Comment:
@kou the only thing that I am not entirely sure whether is required or not in order to run verification is the integration tests that would be run here. Should we add those to some extra checks?
Do you think there is something we can strip from the current `TEST_SOURCE`?

Reply:
I think that we should keep running integration test with `TEST_SOURCE=1`.
Because I think that it's included in " compile as provided, and test the result".